client/components/select-dropdown/index.jsx

```diff
@@ -48,49 +52,27 @@ class SelectDropdown extends Component {
 
 	static defaultProps = {
 		options: [],
-		onSelect: () => {},
-		onToggle: () => {},
+		onSelect: noop,
+		onToggle: noop,
 		style: {},
 	};
 
 	static instances = 0;
```

Comment:
Not something that should be fixed in this PR or any time soon, but I think it's worth noting. If we're ever going to render this component on the server, this can potentially break since this value will be shared between all clients accessing the same server instance.

Reply:
That's a good point: the generated IDs will be different on server and client. Thanks for linking that React issue! As most other people who commented there, all we need the ID for is to link elements together with various ARIA attributes.

I think managing IDs with React Context would solve the issue even in Async mode. I don't immediately see how Async mode could break it. Initial rendering on server and client is always sync and only the later updates can be async, preempt and cancel each other etc.